generic/tests/cfg/unattended_install_reboot_driftfix.cfg

```diff
@@ -0,0 +1,58 @@
+- unattended_install_reboot_driftfix:
+    virt_test_type = qemu libvirt
+    type = unattended_install_driftfix
+    no WinXP Win2000 Win2003 WinVista
+    only Windows
+    extra_params += "-global kvm-pit.lost_tick_policy=delay"
+    backup_image_after_testing_passed = yes
+    start_vm = no
+    kill_vm = yes
+    kill_vm_gracefully = yes
+    kill_vm_on_error = yes
+    shutdown_cleanly = yes
+    shutdown_cleanly_timeout = 120
+    reboot_method = shell
+    force_create_image = yes
+    guest_port_unattended_install = 12323
+    kernel = vmlinuz
+    initrd = initrd.img
+    inactivity_watcher = error
+    inactivity_treshold = 1800
+    image_verify_bootable = no
+    image_copy_on_error = no
+    lowest_mem = 512
```

Comment:
This is a obsolete parameter,  the one to limit guest minimal mem is `vm_mem_minimum`, different guest os have different setting in guest-os.